src/users/mod.rs

```diff
@@ -0,0 +1,77 @@
+mod types;
+use anyhow::{Context, Result};
+use types::*;
+
+use super::Zitadel;
+
+impl Zitadel {
+	/// Crates a new human user. [Docs](https://zitadel.com/docs/apis/resources/user_service_v2/user-service-add-human-user)
+	pub async fn create_human_user(
+		&mut self,
+		user: HumanUser,
+		email_verification: bool,
+		phone_verification: bool,
+	) -> Result<String> {
+		let mut request_data = CreateHumanUserRequest::from(user);
+
+		request_data.email.is_verified = email_verification;
+		if let Some(phone) = request_data.phone.as_mut() {
+			phone.is_verified = phone_verification;
+		}
+
+		let request = self
+			.client
+			.post(self.make_url("v2/users/human"))
+			.json(&request_data)
+			.build()
+			.context("Error building create_human_user request")?;
+
+		tracing::debug!("req: {request:?}");
+
+		self.send_request(request).await
+	}
+}
+
+#[cfg(test)]
+mod test {
+	#![allow(clippy::expect_used)]
+	use std::path::Path;
+
+	use anyhow::Result;
+	use url::Url;
+
+	use super::*;
+
+	#[tokio::test]
+	async fn create_human_user() -> Result<()> {
+		let service_account_file = Path::new("tests/zitadel/service-user.json");
+		let url = Url::parse("http://localhost:8080")?;
+
+		let mut zitadel = Zitadel::new(url, service_account_file.to_path_buf()).await?;
+
+		let user = HumanUser {
+			organization_id: Some("284633323593007107".to_owned()),
+			first_name: "John".to_owned(),
+			last_name: "Doe".to_owned(),
+			preferred_username: "j.doe".to_owned(),
+			email: "j.doe@doamin.com".to_owned(),
+			external_id: "some_id".to_owned(),
+			phone: Some("34666777888".to_owned()),
+			enabled: true,
+			idp_id: None,
+		};
+
+		// Note: If phone or email isVerified are not set to true the creation of the
```

Comment:
At least one auth method must be set up, otherwise Zitadel refuses with the v2 API. As it stands you're not configuring any others, but email/phone verification *is* an auth method.

Zitadel have implemented a way to delay the whole auth method thing with invite links, which we can maybe use to circumvent this: https://github.com/zitadel/zitadel/issues/8310#issue-2410579046